tests/test_dcos_e2e/backends/docker/test_docker.py

```diff
@@ -549,3 +549,61 @@ def test_one_master_host_port_map(self) -> None:
                     if host_ports[0]['HostPort'] == '8000':
                         port_map_found = True
             assert port_map_found
+
+
+class TestDevices:
+    """
+    Tests for setting device mapping on master or agent Docker containers.
+    """
+
+    def test_default(self) -> None:
+        with Cluster(
+            cluster_backend=Docker(),
+            masters=1,
+            agents=0,
+            public_agents=0,
+        ) as cluster:
+            (master, ) = cluster.masters
+            container = _get_container_from_node(node=master)
+            devices = container.attrs['HostConfig']['Devices']
+            assert devices is None
+
+    def test_master_devices(self) -> None:
```

Comment:
I think let's have one test which tests all three new parameters - that way, we can say "`master_devices` does not interfere with agents".